ecommerce/frontend/src/components/profile/ProfileDetails.js

```diff
@@ -0,0 +1,23 @@
+import './ProfileDetails.less'
+import { Link } from 'react-router-dom'
+
+export const ProfileDetails = () => {
+    return (
+        <div className="profile-details">
+            <ul>
+                <li>
+                    {' '}
+                    <Link to="/profile/change-password">Change Password</Link>{' '}
+                </li>
+                <li>
+                    {' '}
+                    <Link to="/profile/purchases">Purchase History</Link>{' '}
+                </li>
+                <li>
+                    {' '}
+                    <Link to="/profile/messages">Messages</Link>{' '}
```

Comment:
Please remove the `{' '}`

Also I would suggest using vertical AntD tabs for this instead. It will look much better in my opinion. https://ant.design/components/tabs/#components-tabs-demo-position

![image](https://user-images.githubusercontent.com/13124263/99870692-43820900-2be6-11eb-94ad-8c680284c1c3.png)